rgw/v2/tests/s3_swift/reusables/bucket_notification.py

```diff
@@ -451,7 +451,7 @@ def apply(self, bucket_name, events):
         log.info(f"get bucket notification for bucket : {bucket_name}")
         get_bucket_notification(self.rgw_s3_client, bucket_name)
 
-    def verify(self, bucket_name):
+    def verify(self, bucket_name, config):
```

Comment:
@anrao19 
please remove this config parameter from here
as config is stored as class variable in the init